src/components/cast/cast.js

```diff
@@ -73,6 +73,7 @@ class Cast extends Component {
    */
   render(props: any): ?React$Element<any> {
     if (props.isCasting || props.isCastAvailable) {
+      const className = props.isCasting ? `${style.castButton} ${style.castButtonActive}` : style.castButton;
```

Comment:
```suggestion
      const className = props.isCasting ? [style.castButton, style.castButtonActive].join(' ') : style.castButton;
```
we used to write it like this

Reply:
I think template string is more readable than using join()

Reply:
better to be aligned but not critical

Reply:
I care more about readability :)